.teamcity/Gradle_Check/configurations/GradleBuildConfigurationDefaults.kt

```diff
@@ -126,6 +126,7 @@ fun applyDefaults(model: CIBuildModel, buildType: BaseGradleBuildType, gradleTas
     var gradleParameterString = gradleParameters.joinToString(separator = " ")
             .replace(java7Homes[OS.linux]!!, java7HomeParameter)
             .replace(java9Homes[OS.linux]!!, java9HomeParameter)
+            .let { if(daemon) it else it.replace("--daemon", "")}
```

Comment:
I would prefer adding `--no-daemon` on our own, so we are not affected if Teamcity decides to change the default. WDYT?

Reply:
I also considered this way, the only disadvantage I can think of is that `--daemon --no-daemon` would confuse users.

Reply:
Not to mention `--daemon --no-daemon -Dorg.gradle.daemon=false`... these 3 parameters appearing together would drive us crazy...

Reply:
I meant that, instead of removing `--daemon`, we should remove `--daemon` and add `--no-daemon`.

I agree with you that we shouldn't have `--daemon --no-daemon`.

Reply:
Done.

Reply:
So much better now!